File: Examples/iOS/Sources/Palette.swift
import SwiftUI

/// Shared colours so the example screens stay in sync.
enum Palette {
    /// Background of the dashboard slider tiles, and the light end of the Sleep gradient.
    static let panelGray = Color(white: 0.80)
    /// The editorial headline purple, and the dark end of the Sleep gradient.
    static let headlinePurple = Color.indigo
    /// The main blue of the daytime sky gradient, and the Focus Flow tile.
    static let daySkyBlue = Color(red: 0.36, green: 0.71, blue: 0.98)
    /// Deep navy ink — readable text/plot on the light daySkyBlue Focus tile.
    static let focusInk = Color(red: 0.06, green: 0.14, blue: 0.36)
}

File: android/example/src/main/kotlin/com/gifcommit/goldengrids/example/Palette.kt
package com.gifcommit.goldengrids.example

import androidx.compose.ui.graphics.Color

/** Shared colours so the example screens stay in sync — mirrors Examples/iOS/Palette.swift. */
object Palette {
    /** Background of the slider tiles, and the light end of the Sleep gradient. */
    val panelGray = Color(0xFFCCCCCC)
    /** The editorial headline purple, and the dark end of the Sleep gradient (iOS indigo). */
    val headlinePurple = Color(0xFF5856D6)
    /** The main blue of the daytime sky gradient, and the Focus Flow tile. */
    val daySkyBlue = Color(0xFF5CB5FA)
    /** Deep navy ink — readable text/plot on the light daySkyBlue Focus tile. */
    val focusInk = Color(0xFF0F245C)
}

/** SwiftUI-style Color(red:green:blue:) from 0..1 components. */
fun rgb(r: Double, g: Double, b: Double): Color = Color(r.toFloat(), g.toFloat(), b.toFloat())
